tests/test_bgp_af.rb

```diff
@@ -828,7 +1001,14 @@ def network_cmd(af, dbg)
                  "#{dbg} Test 3. Restore the removed networks")
 
     # Test: Change route-maps on existing networks
-    should = master.map { |network, rm| [network, "#{rm}_55"] }
+    if platform == :ios_xr
+      ['rtmap1_55', 'rtmap2_55', 'rtmap3_55', 'rtmap5_55', 'rtmap6_55', 'rtmap7_55'].each do |policy|
+        config("route-policy #{policy}", 'end')
```

Comment:
Should you be adding a similar 'no route-policy...' to clean up at the end of the test?